sale_force_invoiced_qty/models/sale_order_line.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    force_invoiced = fields.Float()
```

Comment:
Also you should add help to such fields because their behaviour may look weird for a user who haven't seen the module doc.
```python
help="This amount will be deducted from quantity to invoice. \nquantity to invoice = delivered - invoiced - force invoiced"
```